CONTRIBUTING.md

```diff
@@ -206,8 +206,8 @@ Mainly for markup languages.
 
 ```query
 @markup.strong         ; bold text
-@markup.italic         ; text with emphasis
-@markup.strikethrough  ; strikethrough text
+@markup.italic         ; italic text
+@markup.strikethrough  ; struck through text
```

Comment:
i think keeping it as before is easier to understand

Reply:
But it was wrong...

Reply:
I'd say I'm just more familiar to strikethrough :thinking: 

Both documentation are fine here then.

Reply:
Well, "strikethrough" isn't really an adjective (like "bold" and "italic"), but I'll let that slide...

(How about "struck-through"? Is that better/more understandable?)